portfolio/src/main/java/com/google/sps/data/PropertyNames.java

```diff
@@ -3,6 +3,7 @@
 // Stores field names as constants.
 // Prevents errors due to typos when writing a property name string.
 public class PropertyNames {
+    public static String ANALYSIS_OBJECT_AS_JSON = "analysisObjectAsJson";
```

Comment:
I think this isn't used anywhere, maybe because you realized I had already made the VIDEO_ANALYSIS_JSON property earlier and just used that. This one can be deleted.